### ISSUE TITLE
Pathological Q wave

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10800]
**Preferred term label:**
Pathological Q wave

**Synonyms**
https://www.ncbi.nlm.nih.gov/medgen/C0429090

**Definition (free text, please give PubMed ID)**
An electrocardiographic finding for the Q wave that is atypical for the duration (greater than or equal to 0.03 seconds) and amplitude (greater than or equal to 1 mm (0.1 mV)) in at least 2 contiguous leads; infarct pattern

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fabry disease, cardiac variant

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Not an expert. Most findings are expressed in terms of QRS complex. Is this truly a distinct finding? Is there a good PMID? In the cited article, a pathological U wave (but not Q wave) is described: https://pmc.ncbi.nlm.nih.gov/articles/PMC4040873/

**Comment by MickeySegal** [comment]
Some citations:
- https://pubmed.ncbi.nlm.nih.gov/23242106/
- https://pubmed.ncbi.nlm.nih.gov/19027585/
- https://www.ncbi.nlm.nih.gov/books/NBK83297/ "EKG: dysmorphic notched P-waves, complete or incomplete right BBB or incomplete left BBB, & Q waves in lateral leads"
- https://pubmed.ncbi.nlm.nih.gov/31778001/

There is a UMLS code for "Pathological Q wave" at https://www.ncbi.nlm.nih.gov/medgen/C0429090

**Comment by pnrobinson** [comment]
@MickeySegal 
adding
HP_0020232
Abnormal Q wave, syn  Pathological Q wave
